variants/esp32s3_powerfeather/pins_arduino.h

```diff
@@ -0,0 +1,53 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+
+#define USB_VID            0x303A
+#define USB_PID            0x81BB
+#define USB_MANUFACTURER   "PowerFeather"
+#define USB_PRODUCT        "ESP32-S3 PowerFeather"
+#define USB_SERIAL         ""
+
+#define EXTERNAL_NUM_INTERRUPTS 46
+#define NUM_DIGITAL_PINS        23
+#define NUM_ANALOG_INPUTS       13
+
+#define analogInputToDigitalPin(p)  (((p)<20)?(analogChannelToDigitalPin(p)):-1)
+#define digitalPinToInterrupt(p)    (((p)<48)?(p):-1)
+#define digitalPinHasPWM(p)         (p < 46)
```

Comment:
This part is no longer needed to be defined in the pins_arduino.h file for each variant.
It is defined globally for each SoC. Please remove it to prevent warnings.